de/model.jl

```diff
@@ -127,12 +164,12 @@ function costG!(G, pIn, exp_data)
     end
 
     # Regularization
-    @. G[1:6889] += 10 * (0.01 * sign(pIn[1:6889]))
+    @. G[1:6889] += 1000 * (0.01 * sign(pIn[1:6889]))
     w = reshapeParams(pIn)[1]
     T₀ = w' * w - I
-    temp = 10 * vec(2 / norm(T₀) * w * T₀)
+    temp = 10000 * vec(2 / norm(T₀) * w * T₀)
     @. G[1:6889] += temp
-
+    G += Zygote.gradient(costEigvals, pIn)[1]
```

Comment:
I bet it's coming from the gradient calculation. Try surrounding the Zygote call here with `real.()` to only keep the real component.

Reply:
Same error.. I also managed to get the same error outside of `runOptim` just calling Zygote.gradient on its own

Reply:
Try after the change I just made.

Reply:
Seems to have worked! I can work on resolving merge conflicts